doc/Extensions/Services.md

```diff
@@ -25,6 +25,7 @@ $config['nagios_plugins']   = "/usr/lib/nagios/plugins";
 ```
 
 This will point LibreNMS at the location of the nagios plugins - please ensure that any plugins you use are set to executable.
+Usaully: chmod +x /usr/lib/nagios/plugins
```

Comment:
Change usually to "For example" an put this in a code block.

Also, wouldn't you want `chmod +x /usr/lib/nagios/plugins/*`?

Reply:
oops yes, I will change that.